libs/platform/experience/src/routes.ts

```diff
@@ -1,12 +1,12 @@
 import { TokenResolver } from '@spryker-oryx/core';
 import { resolve } from '@spryker-oryx/di';
-import { RouterService, RouteType } from '@spryker-oryx/router';
+import { RouteType } from '@spryker-oryx/router';
 import { RouteConfig } from '@spryker-oryx/router/lit';
 import { html, TemplateResult } from 'lit';
 import { map, take } from 'rxjs';
 import 'urlpattern-polyfill';
 
-export const defaultExperienceRoutes: RouteConfig[] = [
+export const defaultExperienceLatestRoutes: RouteConfig[] = [
```

Comment:
I'm wondering about just naming it `experienceRoutes`, and use it in latest feature instead of the old one.